### ISSUE TITLE
Adding or deleting Macros to the macro bar causes the card panel to expand

### ISSUE THREAD

**Issue by hirumatto** [spacemandev-git/fvtt-card-support#48]
I have the swade system and card support installed, no other modules installed. Whenever I add a new macro or delete an existing macro from the macro bar the card panel expands. Just seems to be a weird bug.

**Comment by sharkbruhaha** [comment]
Yes, seeing this too, and also running macros, even on other players it makes everyone's pop back up
